src/hooks/Circles/usePrivatePosts.ts

```diff
@@ -81,6 +82,55 @@ export function useInfinitePrivatePosts(userId: string) {
   });
 }
 
+export function usePrivatePostsForUsers(userIds?: string[]) {
+  const { graphQLClient } = useGraphQLClient();
+  const { accountHeaders } = useActiveAccount();
+  const { data } = useUser();
+
+  const queryPosts = async (userId: string) => {
+    const variables = {
+      userIds: [data?.id, userId],
+      filter: {
+        order: 'NEWEST',
+      },
+    };
+
+    return graphQLClient.request<PrivatePostsData>(
+      privatePostsPageInfoQueryDocument,
+      variables,
+      accountHeaders,
+    );
+  };
+
+  return useQueries(
+    (userIds ?? []).map((userId) => {
+      return {
+        queryKey: ['privatePosts', userId],
+        queryFn: () => queryPosts(userId),
+        enabled: !!accountHeaders?.['LifeOmic-Account'],
+        refetchInterval: 10000,
```

Comment:
Currently the only way badge notifications get displayed and cleared...we can do something about the cleared but the displayed requires polling.